src/calendar/tests/CalendarHighlight.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react/addons';
+import CalendarHighlight from '../CalendarHighlight';
+
+const TestUtils = React.addons.TestUtils;
+
+describe('The CalendarHighlight Component', function () {
+  beforeEach(function () {
+    this.spyCx = spyOn(CalendarHighlight.prototype.__reactAutoBindMap, 'cx').and.returnValue('my-class');
+
+    var shallowRenderer = TestUtils.createRenderer();
+    shallowRenderer.render(<CalendarHighlight pending={true} modifier='test'/>);
+    this.renderedComponent = shallowRenderer.getRenderOutput();
+  });
+
+  it('should render the right element', function () {
+    expect(this.renderedComponent.type).toBe('div');
+    expect(this.spyCx).toHaveBeenCalledWith({
+      states: {
+      },
+      modifiers: {
+        test: true,
+      },
+    });
+    expect(this.renderedComponent.props.className).toEqual('my-class');
```

Comment:
I wonder if we should just test the end result here, that the DOM node has the expected classes associated with it when rendered - rather than the internal functions being called in the right way. I think this might produce a less brittle test as a result - thoughts? :)